requirements.txt

```diff
@@ -10,3 +10,4 @@ dill
 flask
 flask-cors
 matplotlib
+pandas

```

Comment:
Pandas is included with Anaconda. It should be installed with `bash Anaconda3-4.3.0-Linux-x86_64.sh -b`.